### DISCUSSION TITLE
Verdacio docker and nginx ssl, error HTTPS request has been blocked

### DISCUSSION THREAD

**Discussion by Tsunani** [verdaccio/verdaccio#3250] [Q&A] [answered]
If I use recommended conf, page not load:

```
location /verdaccio/ {
  proxy_set_header X-Real-IP $remote_addr;
  proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
  proxy_set_header Host $host;
  proxy_set_header X-NginX-Proxy true;
  proxy_pass http://localhost:6666;
  proxy_redirect off;
}
```
And get some errors:

`Mixed Content: The page at 'https://my.site/verdaccio/' was loaded over HTTPS, but requested an insecure script 'http://my.site/verdaccio/-/static/main.f6e570dfe38870ab429f5.js'. This request has been blocked; the content must be served over HTTPS.`

But HTTP work fine.
Pls help solve the problem.

**Comment by edclement** [comment, marked as answer]
The script tags embedded in the HTML page returned from Verdaccio are using `http://`. Try setting a [`url_prefix`](https://verdaccio.org/docs/configuration/#url-prefix)

**Reply by Tsunani** [reply]
In my configuration `url_prefix: /verdaccio/` 
I think the problem is somewhere else

**Reply by edclement** [reply]
Sorry, looking at the docs (same link I provided), it looks like you should set the `VERDACCIO_PUBLIC_URL` env variable to `https://my.site`. and leave the `url_prefix` as is.

**Reply by Tsunani** [reply]
Yep it work.
Thank you so much =)

**Reply by aafnnp** [reply]
cannot open config file /root/.config/verdaccio/config.yaml: YAMLException: can not read a block mapping entry; a multiline key may not be an implicit key (114:11)

![image](https://user-images.githubusercontent.com/12791216/181665053-751e6b30-d33c-400b-bc03-9011389624f7.png)

**Reply by edclement** [reply]
Switch that `=` to a colon. What you have is not valid YML.
```
VERDACCIO_PUBLIC_URL: https://blah
```

**Reply by aafnnp** [reply]
> Switch that `=` to a colon. What you have is not valid YML.
> 
> ```
> VERDACCIO_PUBLIC_URL: https://blah
> ```

thought should change the default style in config.yaml.
